### PR TITLE
fix: add `escape` to `CSS`

### PR DESCRIPTION
**Pull request by alan-agius4**
Closes:
https://github.com/Microsoft/TypeScript/issues/21090

### CODE REVIEW

File: baselines/dom.generated.d.ts [hunk 1]
```diff
@@ -1819,9 +1819,12 @@ declare var CryptoKeyPair: {
 };
 
 interface CSS {
```
Comment (alan-agius4) [inline baselines/dom.generated.d.ts]: I'd expect that something like the below to be the generated output

```ts

interface CSS {
    supports(property: string, value?: string): boolean;
    escape(value: string): string;
}

declare var CSS: CSS;
```

However I cannot seem to make it like that.

**Comment by HolgerJeromin** [comment]
This is still a draft. [Really ready](https://github.com/Microsoft/TSJS-lib-generator#w3c) to be included?

File: baselines/dom.generated.d.ts (hunk 1 above)
Comment (mhegazy) [reply baselines/dom.generated.d.ts]: the generation script is not handling added static properties correctly.. i think i have a fix for that.

**Comment by mhegazy** [comment]
Should be fixed by https://github.com/Microsoft/TSJS-lib-generator/pull/383

**Pull request closed without merging**